whitesource/util.py

```diff
@@ -312,3 +287,126 @@ def scan_artifact_with_white_src(
         # TODO save scanned commit hash or tag in project tag show scanned version
         # version for agent will create a new project
         # https://whitesource.atlassian.net/wiki/spaces/WD/pages/34046170/HTTP+API+v1.1#HTTPAPIv1.1-ProjectTags
+
+
+def _scan_artifact(
+    artifact,
+    extra_whitesource_config,
+    white_src_client,
+    len_artifacts: int,
+    get: typing.Callable,
+    append: typing.Callable,
+):
+    clogger = dso.util.component_logger(__name__)
+    clogger.info(f'scanning {get()}/{len_artifacts}')
+    append(1)
```

Comment:
in case you simply increment a counter, rather name the function `increment` or sth. similar. append suggests to me that something is appended to e.g. a list